build.gradle

```diff
@@ -49,10 +48,25 @@ compileTestKotlin {
     kotlinOptions.jvmTarget = "1.8"
 }
 
+
+markdownToHtml {
+    sourceDir file("$projectDir/docs")
```

Comment:
Turns out that I can move the `CHANGELOG.md` file into `docs` dir with the description file as I was trying to create a script to copy the `CHANGELOG.md` file into a dir and use it as a source, but found out that I can also turn the description altogether which will result in a simpler solution.